rest/connector-generator/src/main/java/io/syndesis/connector/generator/swagger/SwaggerConnectorGenerator.java

```diff
@@ -48,67 +44,77 @@
 import io.syndesis.model.connection.ConfigurationProperty;
 import io.syndesis.model.connection.ConfigurationProperty.PropertyValue;
 import io.syndesis.model.connection.Connector;
-import io.syndesis.model.connection.ConnectorTemplate;
 import io.syndesis.model.connection.ConnectorSettings;
+import io.syndesis.model.connection.ConnectorTemplate;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import static io.syndesis.connector.generator.swagger.DataShapeHelper.createShapeFromModel;
 import static io.syndesis.connector.generator.swagger.DataShapeHelper.createShapeFromResponse;
 
+import static org.apache.commons.lang3.StringUtils.trimToNull;
+
 @SuppressWarnings("PMD.ExcessiveImports")
 public class SwaggerConnectorGenerator extends ConnectorGenerator {
 
     private static final DataShape DATA_SHAPE_NONE = new DataShape.Builder().kind("none").build();
 
-    /* default */ static class PropertyData {
-        private final String defaultValue;
-
-        private final String description;
-
-        private final String displayName;
-
-        private final String[] tags;
-
-        /* default */ PropertyData(final String displayName, final String description, final String defaultValue,
-            final String... tags) {
-            this.displayName = displayName;
-            this.description = description;
-            this.defaultValue = defaultValue;
-            this.tags = tags;
-        }
-    }
+    private static final Logger LOG = LoggerFactory.getLogger(SwaggerConnectorGenerator.class);
 
     @Override
     public Connector generate(final ConnectorTemplate connectorTemplate, final ConnectorSettings connectorSettings) {
         final Connector connector = basicConnector(connectorTemplate, connectorSettings);
 
-        final String specification = requiredSpecification(connectorSettings);
-
-        return configureConnector(connectorTemplate, connector, specification);
+        return configureConnector(connectorTemplate, connector, connectorSettings);
     }
 
     @Override
     public Connector info(final ConnectorTemplate connectorTemplate, final ConnectorSettings connectorSettings) {
         return basicConnector(connectorTemplate, connectorSettings);
     }
 
-    /* default */ Connector basicConnector(final ConnectorTemplate connectorTemplate,
-        final ConnectorSettings connectorSettings) {
+    /* default */ Connector basicConnector(final ConnectorTemplate connectorTemplate, final ConnectorSettings connectorSettings) {
```

Comment:
any particular reason why these methods are not `private` ? For testing ?

Reply:
I can't see it being used in tests, I guess I intended to. I usually prefer package private methods to private, that could also be a hipster thing...